src/components/PrimaryNav.tsx

```diff
@@ -29,13 +29,19 @@ const StarTheRepo = (): JSX.Element => (
   </div>
 );
 
-const PrimaryNav = (): JSX.Element => {
+const PrimaryNav = async (): Promise<JSX.Element> => {
```

Comment:
This is the error. async is not needed is not ideal. Instead `useEffect`. Here is an example https://github.com/open-sauced/hot/blob/beta/src/components/RepoWrap.tsx#L48-L50

https://docs.react-async.com/

Reply:
Hi Brian! Thank you! I removed async on the component, PrimaryNav, and that error is gone now!

I am still having following 2 errors.
1. the API call is returning 401 status.
2.  npm run format fails, saying that I need to catch the error, so I added the try and error keywords, but It's still failing the npm run format... Could you please help?
<img width="1388" alt="Screen Shot 2022-10-12 at 3 52 10 PM" src="https://user-images.githubusercontent.com/58411107/195435605-7fa947d0-ea84-4056-b770-8fa0e5c9c16b.png">
<img width="611" alt="Screen Shot 2022-10-12 at 3 51 10 PM" src="https://user-images.githubusercontent.com/58411107/195435613-2fd3e0ff-ec09-40fc-98b7-7da1b1be0a20.png">

Reply:
Following up from my DM. 

1. Check if you are passing the token or jwt in the call to the API - 401 unauthorized makes me assume not.
2. add a .catch to your function - similar to here: https://github.com/open-sauced/hot/blob/b801823659e89ef81f7eb14682901a20275b8990/src/hooks/useVotedRepos.ts#L18-L20

Reply:
Hi Brian, thank you so much for the help during the meeting! I am now getting 200 successfully, but for some reason, when I run npm run format, I am still getting an error saying that 'Promise must be awaited'.
<img width="595" alt="Screen Shot 2022-10-13 at 2 08 22 PM" src="https://user-images.githubusercontent.com/58411107/195674032-3adfdb2b-5c7d-484c-b3d5-1a581f638959.png">
<img width="595" alt="Screen Shot 2022-10-13 at 2 06 40 PM" src="https://user-images.githubusercontent.com/58411107/195674054-ef13a7a7-a2b0-4814-8780-a2ab448bdc2e.png">

<img width="1574" alt="Screen Shot 2022-10-13 at 2 06 50 PM" src="https://user-images.githubusercontent.com/58411107/195674060-ef46a4da-3266-4e52-9be3-a85c85e232ba.png">

Reply:
Hey Janie-lie, this error is as a result of execution of an async function without the await keyword... adding the `await` keyword to the `fetchAuthSection()` function call should fix. And on a second thought, the function should return the api's call response/error and not `console.log` response/error.